airbyte-integrations/connectors/source-chargify/unit_tests/test_source.py

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from unittest.mock import MagicMock
+
+from source_chargify.source import SourceChargify
+
+
+def test_streams(mocker):
+    source = SourceChargify()
+    config_mock = MagicMock()
+    streams = source.streams(config_mock)
+    expected_streams_number = 2
+    assert len(streams) == expected_streams_number
+
+def test_streams_config(mocker):
+    source = SourceChargify()
+    config_mock = {"subdomain": "tst", "page": 1, "per_page": 200}
+    streams_config = source.convert_config2stream_args(config_mock)
+    assert streams_config == {"subdomain":"tst", "page": 1, "per_page": 200}
```

Comment:
Missing new line at the end of the file.

Reply:
do we have a linter for the project?